todo_app/app/controllers/todo_controller.rb

```diff
@@ -0,0 +1,15 @@
+class TodoController < ApplicationController
+  def index
+  	@todos = Todo.all
+  end
+
+  def show
+  	@todo = Todo.find(params[:id])
+  end
+
+  def new
+  	@todo = Todo.new(params[:todo])
+  	@todo.save
+
+  end
+end
```

Comment:
These controller actions are correct, although the naming syntax for your file and class are incorrect.

file: `todos_controller.rb`
class: `TodosController`